### PR TITLE
Modified landing page content for the anime app

### PR DESCRIPTION
**Pull request by AHuangHe**

### CODE REVIEW

File: client/src/pages/index.tsx [hunk 1]
```diff
@@ -14,10 +14,10 @@ const Home: React.FC<{}> = () => {
       maxW="3xl"
```
Comment (jvoucsb) [inline client/src/pages/index.tsx:14]: Looks good but still need to change the link in the GitHub button to this repository according to #41

File: client/next-env.d.ts [hunk 2]
```diff
@@ -1,6 +1,6 @@
-/// <reference types="next" />
-/// <reference types="next/types/global" />
-/// <reference types="next/image-types/global" />
-
-// NOTE: This file should not be edited
-// see https://nextjs.org/docs/basic-features/typescript for more information.
+/// <reference types="next" />
```
Comment (jvoucsb) [inline client/next-env.d.ts:1]: next-end-d.ts

shouldn't be modified

**Review by jvoucsb — APPROVED** [review]
good good good looks excellent !!

**Review by christopherbarnett — APPROVED** [review]
cb - youve done an absolutely stellar job, keep up the fantastic work, lgtm lgtm lgtm

**Pull request merged**